actioncable/lib/action_cable/engine.rb

```diff
@@ -2,11 +2,13 @@
 require "action_cable"
 require "action_cable/helpers/action_cable_helper"
 require "active_support/core_ext/hash/indifferent_access"
+require "action_dispatch/routing/inspector"
+require "action_cable/route_wrapper"
 
 module ActionCable
   class Railtie < Rails::Engine # :nodoc:
     config.action_cable = ActiveSupport::OrderedOptions.new
-    config.action_cable.url = '/cable'
+    config.action_cable.mount_path = '/cable'
```

Comment:
That seems terribly convoluted to me and in any case, the setup doesn't flow either. Having to set mount_path = nil to configure a url for an out-of-process setup is nasty. Gotta do something better than this